stdlib/kokorawr/kokorawr/functions/services/scores.js

```diff
@@ -0,0 +1,27 @@
+scores = {}
+
+/**
+* Leaderboard information for teams!!!
+* @param {boolean} post if you're trying to store information
+* @param {object} store information to store. Format should be {name: str, info: gameInfo}
+* @returns {object} {<game_name>: [red_score, blue_score]}
+*/
+module.exports = function scores(post=false, store={}, context, callback) {
```

Comment:
@anatasiavela this is a bug... this should be more like it was in the exsmiley/... files because the scores function overrides the scores object.

This error is present in all of the other service functions too

Reply:
```
module.exports = (post=false, store={}, context, callback) => {
```
That's a fix that will work for this line

Reply:
fixed 👍